src/app/content/studyGuides/components/Filters.tsx

```diff
@@ -0,0 +1,43 @@
+import flow from 'lodash/fp/flow';
+import React from 'react';
+import { connect } from 'react-redux';
+import { AppState, Dispatch } from '../../../types';
+import ChapterFilter from '../../popUpElements/ChapterFilter';
+import Filters, { FilterDropdown } from '../../popUpElements/Filters';
+import FiltersList from '../../popUpElements/FiltersList';
+import { setSummaryFilters } from '../actions';
+import * as selectors from '../selectors';
+
+// tslint:disable-next-line:variable-name
+const ConnectedChapterFilter = connect(
+  (state: AppState) => ({
+    locationFilters: selectors.studyGuidesLocationFilters(state),
+    locationFiltersWithContent: selectors.studyGuidesLocationFiltersWithContent(state),
+    selectedLocationFilters: selectors.summaryLocationFilters(state),
+  }),
+  (dispatch: Dispatch) => ({
+    setFilters: flow(setSummaryFilters, dispatch),
+  })
+)(ChapterFilter);
+
+// tslint:disable-next-line:variable-name
+const ConnectedFilterList = connect(
+  (state: AppState) => ({
+    locationFilters: selectors.studyGuidesLocationFilters(state),
+    selectedLocationFilters: selectors.summaryLocationFilters(state),
+  }),
+  (dispatch: Dispatch) => ({
+    setFilters: flow(setSummaryFilters, dispatch),
+  })
+)(FiltersList);
+
+export default () =>
+  <Filters>
+    <FilterDropdown
+      label='i18n:highlighting:filters:chapters'
+      ariaLabelId='i18n:studyguides:popup:filters:filter-by:aria-label'
```

Comment:
Because `label` is literally just `Chapter` , didn't think would be necessary to add an extra one for SG when the filter dropdown has the same name: `Chapter`.

Reply:
The `aria-label` is for the text that comes up in `GA`

Reply:
aria-label is what is read for screen readers for people who can't see well, its used for GA when no `analytics-label` is present